Answers.md

```diff
@@ -1 +1,20 @@
+Describe the biggest difference between .forEach & .map.
 
+forEach acts like a for loop and allows you to interate through each item in a array. Map just orders items into an app.
+
+What is the difference between a function and a method?
+
+function is a piece of code that can perform an action outside of a object. Methods are function inside of objects
+
+What is closure?
+
+Closer has to do with how objects are nested. If they are nested within anothe object they can access the parents objects.
```

Comment:
I think you have pieces of it. The main thing a closure deals with is being able to access variables outside of the function that is referencing it, and the variables inside the function are protected from being referenced by the scope of the function that it is defined in.